package.json

```diff
@@ -23,6 +23,10 @@
     "sequelize": "sequelize",
     "start": "node dist/bin/www.js",
     "start:dev": "nodemon --exec npm run dev",
+    "undo:migrate:test": "NODE_ENV=test sequelize db:migrate:undo:all",
+    "seed:test": "NODE_ENV=test sequelize db:seed:all",
+    "migrate:test": "NODE_ENV=test sequelize db:migrate",
+    "pretest": "NODE_ENV=test npm run undo:migrate:test && npm run migrate:test && npm run seed:test",
```

Comment:
Good job @Andraquin. You don't have to create additional scripts for preparing and clearing your test database, there's already a mechanism handling that for our test. Kindly utilise that.

Reply:
Alright @devdbrandy I have attended to your requested changes